app/src/main/java/com/example/synerzip/recircle_android/ui/ListItemActivity.java

```diff
@@ -153,37 +161,45 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
                 }
             }
         });
-
     }//end onCreate()
 
     @OnClick(R.id.btn_upload_img)
-    public void btnUploadImg(View view){
+    public void btnUploadImg(View view) {
         submitForm();
         HideKeyboard.hideKeyBoard(ListItemActivity.this);
         if (NetworkUtility.isNetworkAvailable(this)) {
             if (getValues()) {
-                Intent intent=new Intent(ListItemActivity.this,UploadImgActivity.class);
+                Intent intent = new Intent(ListItemActivity.this, UploadImgActivity.class);
                 startActivity(intent);
             } else {
                 RCLog.showToast(ListItemActivity.this, getString(R.string.mandatory_dates));
             }
         } else {
             RCLog.showToast(this, getResources().getString(R.string.err_network_available));
         }
-
     }
-public boolean getValues(){
 
-    String strPrice = mEditTxtEnterPrice.getText().toString();
-    String strRental = mEditMinRental.getText().toString();
+    public boolean getValues() {
 
-    if (!strPrice.isEmpty() && !strRental.isEmpty() ) {
-        mItemPrice = Integer.parseInt(mEditTxtEnterPrice.getText().toString().trim());
-        mMinRental = Integer.parseInt(mEditMinRental.getText().toString().trim());
-        return true;
+        String strPrice = mEditTxtEnterPrice.getText().toString();
+        String strRental = mEditMinRental.getText().toString();
+
+        if (!strPrice.isEmpty() && !strRental.isEmpty()) {
+            mItemPrice = Integer.parseInt(mEditTxtEnterPrice.getText().toString().trim());
+            mMinRental = Integer.parseInt(mEditMinRental.getText().toString().trim());
+            if(mDiscounts.getDiscount_for_days()!=0) {
```

Comment:
Instead of hardcoding discounts logic, make your code scalable. Calculate the discount runtime as per the value